projects/gps/src/main/kotlin/gps/GpsSatelliteMessage.kt

```diff
@@ -1,11 +1,15 @@
 package gps
 
+import units.Angle
+import units.Nanodegree
+import units.Quantity
+
 /**
  * A satellite message.
  *
  * @property id the satellite ID (from 1 to 32)
- * @property elevation the elevation in degrees
- * @property azimuth the azimuth measurement in degrees
+ * @property elevation the elevation
+ * @property azimuth the azimuth
  * @property signalRatio the signal-to-noise ratio in dBHz
  */
-data class GpsSatelliteMessage(val id: Int, val elevation: Int?, val azimuth: Int?, val signalRatio: Int?)
+data class GpsSatelliteMessage(val id: Int, val elevation: Quantity<Angle, Nanodegree>?, val azimuth: Quantity<Angle, Nanodegree>?, val signalRatio: Int?)
```

Comment:
`val signalRatio: Int?` should be converted to `Quantity<SignalNoiseRatio, Decibel>`